packages/terra-clinical-item-view/lib/_mixins.scss

```diff
@@ -0,0 +1,21 @@
+@mixin terra-text-body {
+  font-size: 1.1428571428571428rem;
+  line-height: 1.4285714285714286rem;
+
+  @media screen and (min-width: 760px) {
+    font-size: 1rem;
+  }
+}
+
+@mixin terra-x-small-heading {
+  font-size: 1.1428571428571428rem;
+  line-height: 1.4285714285714286rem;
+}
+
+@mixin terra-text-truncate {
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  word-break: normal;
+  word-wrap: normal;
+}
```

Comment:
Should these mixin names be scoped one level deeper just to make sure it doesn't override anything in the future?
```terra-clinical-text-truncate```